app/controllers/contact_attempts_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# ContactAttemptsController: for keeping track of contact attempts over time
+class ContactAttemptsController < ApplicationController
+  before_action :authenticate_user!
+
+  # Create a new history route; this is used to create comments on subjects.
+  def create
+    redirect_to root_url unless current_user.can_create_subject_contact_attempt?
+
+    permitted_params = params.permit(:patient_id, :successful, :note)
+    patient_id = params.require(:patient_id)
+    successful = permitted_params[:successful]
+    note = permitted_params[:note]
+
+    ContactAttempt.create!(patient_id: patient_id,
+                           user: current_user.id,
```

Comment:
This throws an error right now when trying to create a contact attempt - should be `user_id`

Reply:
thanks, fixed here: https://github.com/SaraAlert/SaraAlert/pull/517/commits/d3c1b38e3292c148e183fe00e8dccfb8b81ad227#diff-f5555dc1b7cce5f9da97344de552aa2c17d262d58f826c707ac3fed0a111327aR17